### ISSUE TITLE
[Bug]: pymysql execute "\n" or ";" or "  ", The current connection is blocked.

### ISSUE THREAD

**Issue by tianyahui-python** [matrixorigin/matrixone#6686]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

test.py:
```
import pymysql

def execute():
    db = pymysql.connect(host='localhost', port=6001, user='dump', password='111')
    cur = db.cursor()
    cur.execute('    ')
    cur.execute(';')
    cur.execute('\n')
```

python3 test.py

**Comment by tianyahui-python** [comment]
@iamlinjunhong bug unsolved.

**Comment by iamlinjunhong** [comment]
solved in main, will solve in 0.6
